dataeng/jobs/analytics/WarehouseTransforms.groovy

```diff
@@ -65,14 +65,12 @@ class WarehouseTransforms{
                     }
                 }
                 publishers opsgenie_heartbeat_publisher(allVars)
+                // Writing two different wrappers for credentialBinding won't write two different credentials.
+                // the last wrappers will overwrite will overwrite the previous ones
+                // To overcome this issue write all credentials in a single wrapper
                 wrappers {
                     credentialsBinding {
                         string('OPSGENIE_HEARTBEAT_CONFIG_KEY', 'opsgenie_heartbeat_config_key')
-                    }
-                }
-                wrappers {
-                    timestamps()
```

Comment:
whoops, just noticed this line was removed.  Please add it back.

Reply:
I thought the same but then realized that timestamps() is already added as part of `common_wrappers`.

Reply:
@pwnage101 @HassanJaveed84 So its fine if we don't add it back right ?

Reply:
Correct.